content/agent/kubernetes/host_setup.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Kubernetes Host Setup
+kind: documentation
+further_reading:
+- link: "agent/autodiscovery"
+  tag: "Documentation"
+  text: Docker Agent Autodiscovery
+- link: "agent/kubernetes/host_setup"
+  tag: "documentation"
+  text: "Kubernetes Host Setup"
+- link: "agent/kubernetes/integrations"
+  tag: "documentation"
+  text: "Custom Integrations"
+---
+
+Installing the Agent directly on your host (rather than having the Agent run in a Pod) provides additional visibility into your ecosystem, independent of Kubernetes.
+
+To gather your kube-state metrics:
```

Comment:
I don't think kube-state-metrics and kube-dns should be documented on this page. They're not related to how users install the agent. Happy to chat more about it to find them a better home :)

Reply:
It does indeed make sense to document them here, since their doc is droped/merged to the K8s integration doc in the /integrations section of the doc.